proposals/proposal_1/analysis_options.yaml

```diff
@@ -0,0 +1,13 @@
+# Static analysis allows you to find problems before executing a single line of code.
+# You may customize the Analysis Options by following this guide:
+# https://dart.dev/guides/language/analysis-options
+include: package:pedantic/analysis_options.yaml
+
+analyzer:
+  strong-mode:
+    implicit-casts: false
+    implicit-dynamic: false
+
+linter:
+  rules:
+    prefer_const_constructors: true
```

Comment:
I'm thinking of the following rules:

```yaml
linter:
  rules:
    use_key_in_widget_constructors: true
    always_specify_types: false
```

`prefer_const_constructor` is nice, but it is very heavy handed and can produce errors that cover a lot of code. For an advanced dev, it's easy to see where to put the `const` constructor. For less experience devs, I think it will create friction.

Also, I know you wanted to disable the single-quoted rule, but it seemed odd to use pedantic and then override the rules it defines? This is just a suggestion and I don't have a strong opinion, so if you have two rules here you'd like to use please let me know :)

Reply:
That SGTM